SYCL/Assert/assert_in_simultaneous_kernels.cpp

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: linux
+// REQUIRES: TEMPORARY_DISABLED
```

Comment:
The test was Linux specific. After replace of linux with TEMPORARY_DISABLED we miss this info.
```suggestion
// REQUIRES: linux, TEMPORARY_DISABLED
```

Reply:
Thank you, done